### PR TITLE
Implemented Insert.java and Delete.java

### PR DESCRIPTION
**Pull request by manishsbhoopalam**
#17 
I was assigned to implement these two classes. Most of the methods are very small and simple. But the only method which troubled me was the fetchNext. I tried my level best but the test cases are failing. So please look into this.

### CODE REVIEW

**Review by Rithesh17 — APPROVED** [review]

**Review by Ram-Aditya — APPROVED** [review]

**Review by Priiyam — APPROVED** [review]

**Comment by sbshah97** [comment]
Why has this Pull Request not been merged?

**Comment by Ram-Aditya** [comment]
@salman-bhai We thought we needed to wait for your approval as we had requested a review from your side.
Sorry about that!We `ll go ahead with the merge now then.

**Pull request merged**
